.github/workflows/main.yml

```diff
@@ -266,6 +266,7 @@ jobs:
           path-to-profile: coverage.txt
           flag-name: 'unit'
           parallel: true
+        continue-on-error: true
```

Comment:
nit: add this directly after `if`? Otherwise it looks a bit like an incorrectly indented item...

Reply:
done :)